rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -172,127 +169,159 @@ <h1 class="col-sm-10 col-md-11">{{ record.title }}</h1>
+                    <a href="{{ url_for('records/items.index', path='new', document=record.pid) }}"
+                      type="button" class="btn btn-sm btn-success">
+                      <i class="fa fa-plus-square-o fa-fw"></i> {{ _('Add') }}
                     </a>
-                    <!-- TODO: Style the dropdown header -->
-                    <div class="dropdown-menu" aria-labelledby="dropdownMenu">
-                      <h6 class="dropdown-header">{{ _('Select a Pickup Location') }}</h6>
-                      <div class="dropdown-divider"></div>
-                      {% set locations = item|item_library_pickup_locations %}
-                      {% for location in locations %}
-                        <a class="dropdown-item"
-                          href="{{ url_for('item.patron_request', viewcode=viewcode, item_pid=item.pid, pickup_location_pid=location.pid)}}">
-                            {{ location.get_library().name }}
+                  </h3> 
+                {% endif %}
+              
+            </header>
+            <!-- Display Holdings -->
+            <div class="card">
+              {% for holding in holdings %}
+                {% set items = holding.get_items_filter_by_viewcode(viewcode) %}
+                {% set number_items = items|length %}
+                {% if number_items > 0 %}
+                <!-- Card header -->
+                  <div id="{{ holding.pid }}" class="card-header p-2">
+                    <div class="row">
+                      <div class="col-1 text-center">
+                        <a class="collapse-link" data-toggle="collapse" href="#collapse-{{ holding.pid }}"
+                        aria-expanded="false">
+                          <i class="fa fa-caret-down fa-lg"></i>
                         </a>
-                      {% endfor %}
+                      </div>
+                      <div class="col-2">
+                          {{ holding|holding_location}}
+                      </div>
+                      <div class="col-3">
+                          {{ holding|holding_circulation_category}}
+                      </div>
+                      <div class="col-2">
+                          {% if items|length > 1 %}
+                            {{number_items}} items
+                          {% else %}
+                            {{number_items}} item
+                          {% endif %}
+                      </div>
+                      {% if holding.available %}
+                          {% set avail_text = _('available') %}
+                      {% else %}
+                          {% set avail_text = _('not available') %}
+                      {% endif %}
+                      <div class="col-2 availability" style="display: none;">
+                          {% if number_items > 0 %}
+                            <i class="fa fa-circle text-{{ 'success' if holding.available else 'danger' }}"></i> {{ avail_text }}
```

Comment:
Remove variable `avail_text` and add this `{{ _('available') if holding.available else _('not available') }}`